### ISSUE TITLE
It stopped working after altering 'user.js'

### ISSUE THREAD

**Issue by luccooflinosiii** [Nerixyz/current-song2#523]
Hello,

Everytime I try to add a `user.js` file, the app stops working entirely. I tried redownloading it, but it still doesn't show up on OBS or Chrome. It only started showing up when I re-installed Windows. The logs don't show anything out of the ordinary, I believe:

```
2024-11-10T13:41:38.323788Z  WARN current_song2::config: Couldn't find a single config
2024-11-10T13:41:38.323816Z  WARN current_song2::config: Didn't find any config at any location, creating default one at default location
2024-11-10T13:41:38.334513Z  INFO current_song2: Binding on 127.0.0.1:48457
2024-11-10T13:41:38.335653Z  INFO actix_server::builder: starting 12 workers
2024-11-10T13:41:38.335677Z  INFO actix_server::server: Actix runtime found; starting in Actix runtime
2024-11-10T13:44:26.538901Z  INFO current_song2: Binding on 127.0.0.1:48457
2024-11-10T13:44:26.540106Z  INFO actix_server::builder: starting 12 workers
2024-11-10T13:44:26.540114Z  INFO actix_server::server: Actix runtime found; starting in Actix runtime
2024-11-10T13:46:07.856439Z  INFO current_song2: Binding on 127.0.0.1:48457
2024-11-10T13:46:07.857607Z  INFO actix_server::builder: starting 12 workers
2024-11-10T13:46:07.857621Z  INFO actix_server::server: Actix runtime found; starting in Actix runtime
```

Do I need to delete the cache, or something? 

And a bit unrelated, but how do I make CurrentSong2 display other info (like album name, etc.) in conjunction with what's already in there (currently only the title and artist name)? Thanks in advance!

**Comment by Nerixyz** [comment]
> Do I need to delete the cache, or something?

Your browser might have that file cached. If you shift-click on the reload button, it should reload without a cache. You can also check the network panel in the dev-tools.

> And a bit unrelated, but how do I make CurrentSong2 display other info (like album name, etc.) in conjunction with what's already in there (currently only the title and artist name)?

You can either use [CSS variables](https://currentsong.nerixyz.de/Customization/Theming/#css-variables) or add new elements in a user-script and access the [album info](https://currentsong.nerixyz.de/Customization/User%20Scripts/).

**Comment by luccooflinosiii** [comment]
Gotcha...! Thank you!

It's showing up again! And thanks for pointing me there, I just started learning CSS, and the structure is still pretty weird to me. I tried adding 

```
#song-container.with-album * {
  content: var(--album)
}
```
But it still doesn't show up

**Comment by luccooflinosiii** [comment]
Also, what do you use to track progress? I'm using an open source music player, MusicBee, and I think it tracks it differently.

**Comment by Nerixyz** [comment]
Unfortunately, you can't change the content of existing HTML elements with children (e.g. `<h1 id="title">foo</h1>` can't be changed). But you can insert the pseudo-elements `::before` and `::after`. With these, you can extend the title for example:

```css
#song-container.with-album #title::after {
  content: ' - ' var(--album);
} 
```

![image](https://github.com/user-attachments/assets/3315fe40-826c-4b6f-92c4-74f6e39deeba)

This will also scroll if the title is too long.

---

Alternatively, you can display the album below the artist:

```css
/* Make the container a bit bigger as it contains three lines now */
#song-container.with-image.with-album {
  --max-height: 5.5em;
  --height: 5.5em;
}
#song-container.with-album #mq-subtitle::after {
  content: var(--album);
  font-size: 0.8em;
}
```

![image](https://github.com/user-attachments/assets/727b5d54-95a6-411d-804b-e2eaa8f8791e)

**Comment by luccooflinosiii** [comment]
Yeah, the last one! Tysm! Sorry for making you type it out for me...
